apps/genrsa.c

```diff
@@ -48,7 +48,7 @@ const OPTIONS genrsa_options[] = {
 #endif
 
     OPT_SECTION("Input"),
-    {"3", OPT_3, '-', "Use 3 for the E value"},
+    {"3", OPT_3, '-', "(deprecated) Use 3 for the E value"},
```

Comment:
Guard this with `#ifndef OPENSSL_NO_DEPRECATED_3_0` perhaps?

Reply:
Done